requirements.txt

```diff
@@ -0,0 +1 @@
+selenium (implicit requirement: Chrome)
```

Comment:
requirements.txt files in python projects are usually used with pip. you should be able to do `pip install -r requirements.txt`. And easy way to create this file is also by `pip freeze > requirements.txt`

Also, which version of python are you using?